src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static isLive() {
+    this.alive = this.alive.filter(anim => anim.health > 0);
+  }
+
+  constructor(name, health) {
```

Comment:
You don't take health as the parameter to constructor, the health of new Animal will always be 100
```suggestion
  constructor(name) {
```

Reply:
не побачив! дякую!